packages/sdk-components-react/src/xml-element.ws.ts

```diff
@@ -0,0 +1,22 @@
+import { XIcon } from "@webstudio-is/icons/svg";
+
+import {
+  type WsComponentMeta,
+  type WsComponentPropsMeta,
+} from "@webstudio-is/react-sdk";
+
+import { props } from "./__generated__/xml-element.props";
+
+export const meta: WsComponentMeta = {
+  category: "xml",
+  order: 6,
+  type: "container",
+  icon: XIcon,
```

Comment:
I think this is twitter icon, I will get you a proper one

Reply:
Will provide an icon, because we need one for the navigator even if the component is hidden from components panel